ansible_image_validation/validate-vm-images.yaml

```diff
@@ -17,9 +17,11 @@
         that:
           - "offer_type is defined"
           - "rhel_version is defined"
+          - "build_on_rhui4 is defined"
```

Comment:
where is the corresponding pr for passing this argument to this playbook?

Reply:
https://msazure.visualstudio.com/One/_git/Compute-AzLinux-RedHat-Image/pullrequest/7155504